docs/structure-samples/csharp/csharp.md

```diff
@@ -0,0 +1,53 @@
+## C# and .NET project folder structure
+
+The following guidelines and the folders included, represent the conventional structure for a C#/.NET standard application.
+
+```bash
+~root
+│
+├── .devcontainer.json          # Moved to top
+├── .github/
+│   └── workflows/
+│       ├── workflow1.yml
+│       ├── workflow2.yml
+│       └── ...                     
+│
+├── infra/
+│   ├── main.bicep
+│   ├── main.parameters.bicep
+│   ├── abbreviations.json
+│   └── ...                         
+│
+├── src/
+│   ├── ProjectName.Core/
+│   │   ├── Models/
+│   │   ├── Services/
+│   │   ├── Repositories/
+│   │   └── ...
+│   ├── ProjectName.Web/
+│   │   ├── Controllers/
+│   │   ├── Views/
+│   │   ├── wwwroot/
+│   │   └── ...
+│   ├── ProjectName.Tests/
+│   │   ├── Unit/
+│   │   ├── Integration/
+│   │   └── ...
+│   └── ProjectName.sln
+│
+├── docs/
+│   └── README.md
```

Comment:
Any difference between Readme under root and Readme under docs?

Reply:
Some projects do host extended documentation depending on complexity. Please see https://github.com/Azure-Samples/contoso-real-estate/tree/main/docs
I may add an asterisk to mark it as optional. This may not be checked unless docs folder exists.